layers/+completion/ivy/funcs.el

```diff
@@ -96,6 +96,12 @@
        (spacemacs//gne-init-counsel))
      (pop-to-buffer buf))))
 
+(defun spacemacs//counsel-edit ()
+  "Edit the current search results in a buffer using wgrep."
+  (interactive)
+  (run-with-idle-timer 0 nil 'ivy-wgrep-change-to-wgrep-mode)
```

Comment:
Good question. As far as I could tell, `ivy-occur` cancels execution of this function. I don't know how or why, but I thought it may have something to do with `exit-minibuffer`